packages/vercel/test/redirects.test.js

```diff
@@ -18,6 +18,7 @@ describe('Redirects', () => {
 				},
 				'/blog/[...slug]': '/team/articles/[...slug]',
 				'/Basic/http-2-0.html': '/posts/http2',
+				'/google': 'https://google.com',
```

Comment:
Not officially, but I ran into this when switching out the Cloudflare adapter, where a redirect like the one in the test works. That is what prompted me to create this PR.
Otherwise, I would strongly welcome external redirects becoming officially supported, as this should only require small adjustments, and it feels like the most intuitive approach.